src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -480,22 +480,20 @@ public static void addSub(DLNAMediaSubtitle currentSubTrack, DLNAMediaInfo media
 	@Deprecated
 	// FIXME this is obsolete (replaced by the private method below) and isn't called from anywhere outside this class
 	public static void getFormat(MediaInfo.StreamType streamType, DLNAMediaInfo media, DLNAMediaAudio audio, String value) {
```

Comment:
personally I would rename also this method. Or even completely remove it if it is `deprecated` and unused as described in comment

Reply:
@drakulis no problem.